### DISCUSSION TITLE
Plugin for image hosting in a google cloud bucket/firebase storage bucket?

### DISCUSSION THREAD

**Discussion by pepperpepperpepper** [medusajs/medusa#3180] [Feature Requests]
Are there any plans to implement a plugin like this?

**Comment by breakerh** [comment]
If someone really wants this you could copy the digital ocean bucket plugin and alter it to use cloud storage for firebase.you will only need to alter the digital ocean side. The connection to Medusa is already there.

**Comment by srindom** [comment]
Hi @pepperpepperpepper - we don't have plans to build an integration to Google Buckets. I'd recommend that you take a look at [the S3 plugin](https://github.com/medusajs/medusa/tree/master/packages/medusa-file-s3) and try building it yourself :)
